README.md

```diff
@@ -12,23 +12,29 @@ no run-time dependencies.
 Build
 =====
 
-An already built version is included under build; if you build manually,
-this will be overwritten.
+An already built version for use in browsers is included under build; if
+you build manually, this will be overwritten.
 
-To do a build that generates both uncompressed and minified versions, 
+To do a build that generates both uncompressed and minified versions,
 simply use make. 'make compile' only generates the uncompressed version.
 Both versions are placed in './build'.
 
 Simply include either dice.js or dice.min.js in a script tag, and you're
 off.
 
+For use with node, use the files in the src directory; you should only
+need to `var dice = require("./src/dice.js");`.
```

Comment:
Not strictly true; if you're publishing on npm, it will just be `var dice = require('dice');` (or whatever the package name is).